pkg/qe-tests/cypress/integration/models/plan.ts

```diff
@@ -350,7 +350,7 @@ export class Plan {
         .within(() => {
           cy.get('[data-label="Status"]', { timeout: 3600 * SEC }).should(
             'contain.text',
-            'Idle - Next incremental copy will begin in less than 1 minute'
+            'Idle - Next incremental copy will begin in '
```

Comment:
Why is the message incomplete ?

Reply:
I guess that we need to do several pre-copies, at least 2. So please don't remove that time, leave it or update it to be 5-10 minutes, then put another one with for example 50 minutes. So when cutover will run - we will be sure that at least 2 precopies were done

Reply:
Thanks @sshveta and @ibragins for reviewing the PR.
In warm migration after it completes its first incremental pre-copy the status displays this message and we are using this status as an assertion in our code.
As the nextPrecopyAt timestamp is not static, it is calculating time until the next copy will start and we can't predict what will be the exact time-period it will start and hence I just implemented it just catching the text without the time-period.
As I observed something like this "begin in 2 minutes" and "begin in 59 minutes".

Thanks @ibragins for this suggestion I'll start working on it.